pkg/kubeapiserver/admission/initializer.go

```diff
@@ -62,7 +62,7 @@ type WantsQuotaConfiguration interface {
 // WantsServiceResolver defines a fuction that accepts a ServiceResolver for
 // admission plugins that need to make calls to services.
 type WantsServiceResolver interface {
```

Comment:
I am surprised that these "WantsX" interfaces are defined here rather than in the generic layer, but that's a can of worms for some other time.

Reply:
```bash
$ git grep "type Wants.* interface"
pkg/kubeapiserver/admission/initializer.go:35:type WantsInternalKubeClientSet interface {
pkg/kubeapiserver/admission/initializer.go:41:type WantsInternalKubeInformerFactory interface {
pkg/kubeapiserver/admission/initializer.go:47:type WantsCloudConfig interface {
pkg/kubeapiserver/admission/initializer.go:52:type WantsRESTMapper interface {
pkg/kubeapiserver/admission/initializer.go:57:type WantsQuotaConfiguration interface {
pkg/kubeapiserver/admission/initializer.go:64:type WantsServiceResolver interface {
pkg/kubeapiserver/admission/initializer.go:76:type WantsAuthenticationInfoResolverWrapper interface {
staging/src/k8s.io/apiserver/pkg/admission/initializer/interfaces.go:28:type WantsExternalKubeClientSet interface {
staging/src/k8s.io/apiserver/pkg/admission/initializer/interfaces.go:34:type WantsExternalKubeInformerFactory interface {
staging/src/k8s.io/apiserver/pkg/admission/initializer/interfaces.go:40:type WantsAuthorizer interface {
staging/src/k8s.io/apiserver/pkg/admission/initializer/interfaces.go:46:type WantsScheme interface {
```

Reply:
Hm, maybe some of those aren't generic. I think auth info, service resolver, and rest mapper probably are. Don't fix it in this PR :)